pkg/storaged/nfs-panel.jsx

```diff
@@ -0,0 +1,347 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import React from "react";
+import $ from "jquery";
```

Comment:
We try to avoid jQuery in React components. Here it's only being used to select elements by ID, which can be done equally well with `document.getElementById()` or more generally with `document.querySelector()`.

Reply:
We use it in `$(...).modal('show')` which is not easily replaced, I think.  My plan is to clean up the error dialog together with the rest of the dialog machine.